### PR TITLE
Adds descheduler RN to 4.13 Rns

### PR DESCRIPTION
**Pull request by stevsmit**
Descheduler RNs for 4.13. 

Version(s):
4.13 RNs 

Issue:
https://issues.redhat.com/browse/OCPBUGS-7876

Link to docs preview:
https://58463--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-bug-fixes

Qe not needed

Additional information:

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
Added a comment that I don't think this PR is needed.

Let me know if you disagree, and I can do a real peer review on it.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -963,6 +963,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-kube-scheduler-bug-fixes"]
 ==== Kubernetes Scheduler
 
+* Previously, when the `LifeCycleUtilization` profile was excluded to test namespace filtering, the following error was logged in the Descheduler Operator logs: `belowE0222 12:43:14.331258 1 target_config_reconciler.go:668] key failed with : only namespace exclusion supported with LowNodeUtilization`. Consequently, the Descheduler cluster pod would not start. This issue has been resolved and namespace exclusion now works with the `LifeCycleUtilization` profile. (link:https://issues.redhat.com/browse/OCPBUGS-7876[*OCPBUGS-7876*])
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply release_notes/ocp-4-13-release-notes.adoc:966]: Thanks for letting me know!

**Pull request closed without merging**